src/components/Error.jsx

```diff
@@ -0,0 +1,10 @@
+function PageNotFound() {
+  return (
+    <div className="error-invalid-url-page">
+      <p>Oops there has been an error !</p>
+      <p>Please check your URL</p>
+    </div>
+  );
+}
+
+export default PageNotFound;
```

Comment:
Nice use of a custom error component, but I'd align the component name with the file name